### ISSUE TITLE
sorted_imports rule edge cases (annotations support)

### ISSUE THREAD

**Issue by shuguenot** [realm/SwiftLint#5825]
### New Issue Checklist

- [x] I've Updated SwiftLint to the latest version.
- [x] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

Adding `@preconcurrency` / `@_exported` annotation breaks `sorted_imports` rule :

```swift
@_exported @preconcurrency import AAAAA
@_exported import BBBBB // <- sorted_imports violation is triggered
@_exported import CCCCC
```

```bash
$ swiftlint lint --no-cache
```

### Environment

* SwiftLint version 0.57.0
* Xcode version 16.0
* Installation method used Homebrew
* Configuration file:

```yml
opt_in_rules:
  - sorted_imports
sorted_imports:
  grouping: attributes
```

**Comment by SimplyDanny** [comment]
The suggested order is probably

```swift
@_exported import BBBBB
@_exported import CCCCC
@_exported @preconcurrency import AAAAA
```

according to the `attributes` ordering option, isn't it?

**Comment by shuguenot** [comment]
You're right. Thanks for the help
